apps/omg_watcher/priv/swagger/informational_api_specs.yaml

```diff
@@ -70,7 +70,7 @@ paths:
       produces:
       - application/json
       requestBody:
-        $ref: '#/requests/GetTransactionsOpId'
+        $ref: '#/requests/GetTransactionsAllOpId'
       responses:
         200:
           $ref: '#/responses/GetTransactionsOpId'
```

Comment:
Why this did not change?

Reply:
Both routes `/account.get_transactions` and `/transaction.all` are served by the same controller function, so they should (and now) have the same request / response specs